modules/ossm-rn-new-features.adoc

```diff
@@ -17,13 +17,175 @@ This release adds improvements related to the following components and concepts.
+
+When upgrading to Service Mesh Control Plane 2.3, the existing `istio-cni-node` DaemonSet is not changed, and a new `istio-cni-node-v2-3` DaemonSet is created.
+
+This namespace change does not affect previous releases or any `istio-cni-node` CNI DaemonSet associated with a Service Mesh Control Plane deployed using a previous release.
+
+=== Gateway injection support
+
+This release introduces Service Mesh Gateway injection. Gateway configurations are applied to standalone Envoy proxies that are running at the edge of the mesh, rather than the sidecar Envoy proxies running alongside your service workloads. This enables the ability to customize gateway options. When using gateway injection, you must create the following resources in the namespace where you want to run your gateway proxy: `Service`, `Deployment`, `Role`, and `RoleBinding`.
+
+=== Istio 1.14 Support
+
+{SMProductShortName} 2.3 is based on Istio 1.14, which brings in new features and product enhancements. While many Istio 1.14 features are supported, the following exceptions should be noted:
+
+* ProxyConfig API is supported with the exception of the image field.
+* Telemetry API is a tech preview feature.
+* SPIRE runtime is not a supported feature.
+
+[id="service-mesh-2-3-0-tech-prev_{context}"]
+=== Technology Preview
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+[IMPORTANT]
+====
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see the link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Support Scope].
+====
+
+==== OpenShift Service Mesh Console
+
+This release introduces a Developer Preview version of the OpenShift Service Mesh Console, which integrates the Kiali interface directly into the OpenShift Console. For additional informaiton, see link:https://cloud.redhat.com/blog/introducing-the-openshift-service-mesh-console-a-developer-preview[Introducing the OpenShift Service Mesh Console (A Developer Preview)]
+
+==== Cluster-Wide deployment
+
+This release introduces cluster-wide deployment as a technology preview feature. A cluster-wide deployment contains a Service Mesh Control Plane that monitors resources for an entire cluster. The control plane uses a single query across all namespaces to monitor each Istio or Kubernetes resource kind that affects the mesh configuration. Reducing the number of queries the control plane performs improves performance.
```

Comment:
To explicitly call out the difference between the cluster-wide and the multi-tenant deployment, it might make sense to explain that the multi-tenant approach uses a query *per namespace* for each resource kind.

Reply:
Change made.